index.html

```diff
@@ -184,39 +189,29 @@ <h2 class="h2-size">I'm always interested in hearing about new projects, so if y
               <input type="email" id="mail" name="user_email" placeholder="Email-address" required>
             </div>
             <div>
-              <textarea id="msg" name="user_message" placeholder="Enter text here" rows="6" maxlength="500" required></textarea>
+              <textarea id="msg" name="user_message" placeholder="Enter text here" rows="5" maxlength="500" required></textarea>
             </div>
             <div>
               <button type="submit" class="footer-btn">Get in touch</button>
             </div>
           </form>
```

Comment:
The green border should appear when you focus any `input` or `textarea` in your form, to access this CSS attribute try to use the next one:
```
input:focus {
  border-color: green;
}
```

- [ ] Add green border when you focus any `input` or `textarea` in your contact me form.


![image](https://user-images.githubusercontent.com/60164239/130193355-8636a873-d99b-4c02-b905-745e39a89239.png)

Reply:
Thank you for this review

Reply:
Well solved